examples/prometheus/BUILD

```diff
@@ -0,0 +1,14 @@
+cc_binary(
+    name = "prometheus_example",
+    srcs = [
+        "main.cc",
+    ],
+    linkopts = ["-pthread"],
+    tags = ["ostream"],
+    deps = [
+        "//api",
+        "//examples/common/metrics_foo_library:common_metrics_foo_library",
+        "//exporters/prometheus:prometheus_exporter",
+        "//sdk/src/metrics",
+    ],
+)
```

Comment:
Can we add cmake build too, or else github issue to track if plan to add it separately ?